Project.toml

```
@@ -1,8 +1,9 @@
 name = "StatsModels"
 uuid = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
-version = "0.6.33"
+version = "0.7.0-DEV"
```

Usually we just set the version that will be released (I'm not even sure Pkg accepts letters, which may explain the docs build failure).

```suggestion
version = "0.7.0"
```

---

Actually we should even do this:
```suggestion
version = "1.0.0"
```